packages/node_modules/@ciscospark/internal-plugin-wdm/test/unit/spec/device.js

```diff
@@ -42,9 +42,9 @@ describe('plugin-wdm', () => {
 
       describe('#_updateServiceCatalog', () => {
         it('refreshes the device if serviceHostMap is not defined and ha is enabled', () => {
-          device.services = {
-            validServiceUrl: 'http://example.com/valid-service',
-            anotherServiceUrl: 'http://another.com/another-service'
+          device.serviceHostMap.serviceLinks = {
+            valid: 'http://example.com/valid-service',
```

Comment:
I prefer the less ambiguous `validServiceUrl`. Please see my comments at the beginning.

Reply:
@mattnorris Same reason above, make it compatible and consistent with the new map in the message received from the server side.